lib/plugins/aws/deploy/lib/iam-policy-lambda-execution-template.json

```diff
@@ -1,5 +1,8 @@
 {
   "Type": "AWS::IAM::Policy",
+  "DependsOn": [
+    "IamRoleLambdaExecution"
```

Comment:
@pmuens I noticed this https://github.com/nicka/serverless/blob/c3620e501b02c52327f75f486a431b2186c41edb/lib/plugins/aws/deploy/lib/iam-policy-lambda-execution-template.json#L30 on `master`. Do we need to implement this as well?